src/ResourceManager/Network/Commands.Network/VirtualNetworkGatewayConnection/NewAzureRmIpsecPolicyCommand.cs

```diff
@@ -22,15 +22,15 @@ namespace Microsoft.Azure.Commands.Network
     public class NewAzureRmIpsecPolicyCommand : NetworkBaseCmdlet
     {
         [Parameter(
-            Mandatory = true,
+            Mandatory = false,
             HelpMessage = "The IPSec Security Association (also called Quick Mode or Phase 2 SA) lifetime in seconds")]
-        [ValidateNotNullOrEmpty]
+        [ValidateRange(300, 172799)]
         public int SALifeTimeSeconds { get; set; }
 
         [Parameter(
-            Mandatory = true,
+            Mandatory = false,
             HelpMessage = "The IPSec Security Association (also called Quick Mode or Phase 2 SA) payload size in KB")]
-        [ValidateNotNullOrEmpty]
+        [ValidateRange(1024, int.MaxValue)]
```

Comment:
IPsec Policy is only used by the Put connection call. It would be rejected as the backend would check the SA values and see it was out of bounds. There is no logic change overall.
This PR is meant to address complaints that 1) customers were surprised to see policy get rejected on the Put call and wanted to see the policy get rejected on creating the object instead 2) customer wanted some default values to be set for unspecified SAs